### PR TITLE
Social Anxiety Change

### PR DESCRIPTION
**Pull request by Sparrowbird611**
## About The Pull Request

Removes the "you retreat into yourself" part of social anxiety (it silences you) and increases the stutter chance based on the number of people around you exponentially.

## Why It's Good For The Game

This thing is 1 point trait perk that has one of the most oppressive side effects in the game (silence). This makes it actually worth 1 point, and removes annoying interactions where you have to wait two minutes for the chemist you're talking with to stop panicking.

## Changelog
:cl:
tweak: Social Anxiety no longer silences you. It now increases your stutter chance based on the number of people around you.
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
Just increase the number of quirk points Social Anxiety gives you for taking it. No one's gonna give a shit about some stuttering, and this is gonna become a free quirk point.

**Comment by Sparrowbird611** [comment]
Eh, maybe. If I go that route though, i'm increasing the number of people it takes to make you go mute.

File: code/datums/traits/negative.dm [hunk 1]
```diff
@@ -426,11 +426,8 @@
 		if(H.client)
 			nearby_people++
 	var/mob/living/carbon/human/H = quirk_holder
-	if(prob(2 + nearby_people))
+	if(prob(2 + 2^nearby_people))
 		H.stuttering = max(3, H.stuttering)
-	else if(prob(min(3, nearby_people)) && !H.silent)
-		to_chat(H, "<span class='danger'>You retreat into yourself. You <i>really</i> don't feel up to talking.</span>")
-		H.silent = max(10, H.silent)
```
Comment (ike709) [inline code/datums/traits/negative.dm:433]: I think slightly nerfing the max silence would be a better change than outright removing silence.

Comment (Sparrowbird611) [reply code/datums/traits/negative.dm:433]: I thought of that, but I didn't for two reasons.

A: Silence is definitely one of the most crippling downsides you can have.
B: I'm honestly not sure how to do that.

**Comment by qwertyquerty** [comment]
This is not a favored solution.

**Pull request closed without merging**